ngraph/src/ngraph/op/range.cpp

```diff
@@ -25,6 +25,94 @@
 using namespace std;
 using namespace ngraph;
 
+constexpr NodeTypeInfo op::v4::Range::type_info;
+
+op::v4::Range::Range(const Output<Node>& start,
+                     const Output<Node>& stop,
+                     const Output<Node>& step,
+                     element::Type output_type)
+    : Op({start, stop, step})
+    , m_output_type(output_type)
+{
+    constructor_validate_and_infer_types();
+}
+
+bool ngraph::op::v4::Range::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("output_type", m_output_type);
+    return true;
+}
+
+void op::v4::Range::validate_and_infer_types()
+{
+    auto const_start = as_type_ptr<op::Constant>(this->input_value(0).get_node_shared_ptr());
```

Comment:
Specify the exact version: `opset4::Constant`

Reply:
there is only v0::Constant, not sure if I should specify v0